src/routes/[deviceUID]/dashboard/+page.server.ts

```diff
@@ -39,6 +40,7 @@ export async function load({ params }) {
     .then((responses) => {
       const [envVarResponse, eventsResponse] = responses;
       const serialNumber = envVarResponse.environment_variables._sn;
+      airIndoors = envVarResponse.environment_variables?._air_indoors ?? 0;
```

Comment:
then change this snappy one liner to a more readable `if` statement that sets `airIndoors` to `true` if need be
```suggestion
            if (envVarResponse.environment_variables._air_indoors === 1) {
        airIndoors = true;
      }
```